runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -239,4 +239,14 @@ default boolean swapDecant()
 	{
 		return false;
 	}
+	@ConfigItem(
+		position = 19,
+		keyName = "swapCollect",
+		name = "Collect",
+		description = "Swap Collect with Talk-to for Priestess Zul-Gwenwynig at Zul-Andra."

```

Comment:
This will also swap talk to and collect for the vorkath guy  I guess

Reply:
Or at least it should

Reply:
I made it only for Zulrah because Torfinn also has a Travel option and there’s others, like Miscellania guy.